scrapy/http/response/ftplist.py

```diff
@@ -0,0 +1,18 @@
+"""
+This module implements the FTPListResponse class which stores files
+listed in FTP folders, in a dict structure. Other attributes are typical
+of Response.
+
+See documentation in docs/topics/request-response.rst
```

Comment:
There is no extra documentation in docs/topics/request-response.rst for FTPListResponse.